hw/ip/gpio/data/gpio.prj.hjson

```diff
@@ -20,9 +20,11 @@
       {
         version:            "1.1",
         life_stage:         "L1",
-        design_stage:       "D2S",
+        design_stage:       "D3",
         verification_stage: "V2",
         dif_stage:          "S2",
+        commit_id:          "e8966d18587d5b90d464f2eaf1e380b6b52cef77"
```

Comment:
FYI:
```
$ git show e8966d18587d5b90d464f2eaf1e380b6b52cef77
fatal: bad object e8966d18587d5b90d464f2eaf1e380b6b52cef77
```

Reply:
Is this maybe pointing to some local commit hash?
Also, as long as we're just moving forward (and not backward, were a new version entry needs to be created), I believe our process does not require recording the `commit_id`.